tests/test_geo_utils.py

```diff
@@ -5,58 +5,70 @@
 
 import pytest
 
-from sentinelhub import CRS, BBox, geo_utils
+from sentinelhub import CRS, BBox
+from sentinelhub.geo_utils import (
+    bbox_to_dimensions,
+    bbox_to_resolution,
+    get_image_dimension,
+    get_utm_crs,
+    pixel_to_utm,
+    to_wgs84,
+    transform_point,
+    utm_to_pixel,
+    wgs84_to_pixel,
+    wgs84_to_utm,
+)
 
+BBOX_WGS84 = BBox(((111.644, 8.655), (111.7, 8.688)), CRS.WGS84)
+BBOX_UTM = BBox(((570851.8316965176, 956770.2133183606), (577006.4223865959, 960429.6742984429)), CRS("32649"))
+BBOX_POP_WEB = BBox(((12428153.230124235, 967155.4076988235), (12434387.12160866, 970871.4287682716)), CRS.POP_WEB)
```

Comment:
why do these have an absurd amount of decimal points?
For the UTM BBox you're specifying nanometers!

Reply:
if you're going to use the same bbox just reprojected a few times, then you can't really avoid having lots of decimals. But i think that tests would work better with three independant bboxes (outside of the tests for reprojection).